source/fab/steps/find_source_files.py

```diff
@@ -152,7 +152,7 @@ def find_source_files(config, source_root=None,
     # Fortran, C, and PSyclone
     config.artefact_store.copy_artefacts(output_collection,
                                          ArtefactSet.FORTRAN_BUILD_FILES,
-                                         suffixes=[".f90", ".F90"])
+                                         suffixes=[".f", ".F", ".f90", ".F90"])
```

Comment:
...see, here you _did_ include all possible extensions.  Why didn't you do it above?

Reply:
The find_source_files stage is the the first (or one of the first if your script downloads code first) that runs. It finds all source files, which must include all Fortran files. When the pre-processor steps are done, this step will take all .F/F90 files, preprocess them (and store them in the build directory of the Fab work space). Then it will remove the .F/.F90 files from FORTRAN_BUILD_FILES, and add the new .f/.f90 files instead.

You can find this documented in the section ArtefactStore of writing_config.rst - I tried to explain the 'standard' flow, which should explain this a bit better (and suggestions for improvements are always welcome).